### PR TITLE
Bug 1551850 - move all generated Python into generated/

### PR DESCRIPTION
**Pull request by djmitche**
This also adds some tests that just import various things, to check that imports still work.  Note that the README only suggests `taskcluster.Queue` and `taskcluster.aio.Queue` for example, and never `taskcluster.queue.Queue`.

Bugzilla Bug: [1551850](https://bugzilla.mozilla.org/show_bug.cgi?id=1551850)

### CODE REVIEW

**Comment by djmitche** [comment]
@escapewindow I've also invited you to a team on the TC org that can be flagged for (non-generated) changes to the Python client -- and we can add people other than you :)

File: clients/client-py/README.md [hunk 1]
```diff
@@ -213,6 +213,28 @@ instance.  This is meant to assist those who do not wish to bother figuring out
 how to configure the python logging module but do want debug messages
 
 
+## SlugIDs
+
+To generate slugIds (Taskcluster's client-generated unique IDs), use
+`taskcluster.slugId()`, which will return a unqiue slugId on each call.
```
Comment (escapewindow) [inline clients/client-py/README.md]: sp: unique

**Comment by escapewindow** [comment]
I'm guessing once I fix my node version, that `yarn` inside `clients/client-py` will generate the code?

**Comment by djmitche** [comment]
Close -- `yarn generate` in the root of the repo will generate everything including the Python client.

**Pull request merged**